packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1300,6 +1300,7 @@ The payload includes the following reserved claims:
 
 | Property | Description                                                                                                            | DataType |
 | -------- | ---------------------------------------------------------------------------------------------------------------------- | -------- |
+| auth_time| The time when the authentication occurred, represented in Unix time (seconds).                                         | Integer  |
```

Comment:
This property is already documented in the section about ID Token payload. I think it would be easier if we copied the exact same text to avoid confusion for the develop if they read slightly different phrasings of this claim, as the claim is the exact same claim. Here's how it is phrased for ID Token:

```
The time the end user was authenticated, represented in Unix time (seconds).
```

Reply:
Hi Lars, I updated the description and the payload example. I ended up using `${authorizationServerId}` instead of the GUID in the `iss` claim. Let me know if you want that reverted. Or if you want the other GUIDs changed to a variable instead (i.e. ${clientId}, ${userId}, etc.)